spec/rspec/rails/view_spec_methods_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "spec_helper"
+
+module RSpec::Rails
+  describe ViewSpecMethods do
```

Comment:
I haven't set it just yet, but I'd like to move towards zero monkey patching mode. Could we use `RSpec.describe` for this outter most `describe`? (It's the only one which needs it; inside this block `describe` is available as part of the block's DSL)